www/public/repl/index.html

```diff
@@ -1,39 +1,42 @@
 <!DOCTYPE html>
 
 <html>
-  <head>
-    <title>Roc REPL</title>
-    <link rel="stylesheet" href="/repl/repl.css" />
-  </head>
 
-  <body>
-    <div class="body-wrapper">
-      <section class="text">
-        <h1>The rockin' Roc REPL</h1>
-      </section>
+<head>
+  <title>REPL</title>
+  <link rel="stylesheet" href="/site.css" />
+  <link rel="stylesheet" href="/repl/repl.css" />
+  <link rel="icon" href="/favicon.svg">
+  <!-- Safari ignores rel="icon" and only respects rel="mask-icon". It will render the SVG with
+    fill="#000" unless this `color` attribute here is hardcoded (not a CSS `var()`) to override it.
+  -->
+  <link rel="mask-icon" href="/favicon.svg" color="#7d59dd">
+</head>
 
-      <section class="history">
-        <div class="scroll code">
-          <div id="help-text"></div>
-          <div id="history-text">
-            <div id="loading-message">
-              Loading Roc compiler as a WebAssembly module... please wait!
-            </div>
-          </div>
+<body>
+  <div id="top-bar">
+    <nav><a class="home-link" href="/" title="The Roc Programming Language">roc</a>
+      <div id="top-bar-links"><a href="/tutorial">tutorial</a><a
+          href="https://github.com/roc-lang/roc/tree/main/getting_started">install</a><a href="/repl">repl</a><a
+          href="/builtins/Bool">docs</a></div>
+    </nav>
+  </div>
+  <main>
+    <h1>The rockin’ Roc REPL</h1>
+    <code class="history">
+      <div id="help-text"></div>
+      <div id="history-text">
+        <div id="loading-message">
+          Loading Roc compiler as a WebAssembly module…please wait!
```

Comment:
Maybe a space after the ellipsis?

Reply:
Interesting - I always learned not to have spaces around ellpises! Apparently its' a [Chicago Manual of Style](https://english.stackexchange.com/a/26254) convention. 🤷

Reply:
Oh I see! These slight differences between the dialects of English seem to be endless!